modules/web/src/assets/css/theme/_main.scss

```diff
@@ -26,9 +26,15 @@
 //  If you specify typography styles for the components you use elsewhere, you should delete this line.
 //  If you don't need the default component typographies but still want the hierarchy styles,
 //  you can delete this line and instead use:
-//    `@include mat.legacy-typography-hierarchy(mat.define-legacy-typography-config());`
+//    `@include mat.legacy-typography-hierarchy(mat.define-typography-config());`
+
+/* TODO(mdc-migration): Remove all-legacy-component-typographies once all legacy components are migrated */
 @include mat.all-legacy-component-typographies;
+@include mat.all-component-typographies;
```

Comment:
do we need to remove `@include mat.all-legacy-component-typographies;`
and also the `TODO` comment lines ?

Reply:
Yes, those are added by Angular Material's migration tool and I think these should remain here until we've migrated all components.